### PR TITLE
WIP - Feature/make player full width

### PR DESCRIPTION
**Pull request by sabeurthabti**
### Make audio player full wifth
I can't remember why we made the audio player small and push to one corner. 
### Checklist

- [ ] Unit tests written
- [x] Manually tested
- [x] Prettier & ESLint were run
- [x] New dependencies are included in `package-lock.json`

### Screenshot
screen shot 2018-01-15 at 00 20 18
screen shot 2018-01-15 at 00 19 26

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32930

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32931

**Comment by sabeurthabti** [comment]
maybe we can do something like the quranicaudio player, and have mobile be fullscreen. Just an idea.

**Comment by naveed-ahmad** [comment]
@sabeurthabti this looks beautiful mashAlllah

**Comment by naveed-ahmad** [comment]
@sabeurthabti can't see player on mobile. https://www.youtube.com/watch?v=PpPwGAw2V-w

**Comment by sabeurthabti** [comment]
@naveed-ahmad I will test again, I was testing it in responsive view. I will use a real device iA. thanks for taking time to share the view.

**Comment by mmahalwy** [comment]
This looks awesome! Thoughts about what happens to the Zendesk help button?

**Comment by sabeurthabti** [comment]
the zendesk is still there, not sure why it wasn't showing in the screenshot. I will make look nice on mobile too.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32933

**Comment by sabeurthabti** [comment]
I will create some space so the Zendesk button shouldn't be altered or impact on the player. 

I pushed a few things:
- Add support for keyboard shortcuts
- fixed bug with when clicking ahead of the player to fast forward.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32934

**Comment by sabeurthabti** [comment]
@naveed-ahmad 
screen shot 2018-01-21 at 01 40 01

File: package.json [hunk 1]
```diff
@@ -28,16 +26,38 @@
...
   },
-  "pre-commit": ["test:stylelint", "precommit"],
+  "pre-commit": [
+    "test:stylelint",
+    "precommit"
+  ],
   "lint-staged": {
-    "*.js": ["npm run prettier", "npm run lint:fix", "git add"],
-    "*.scss": ["npm run prettier", "git add"]
+    "*.js": [
```
Comment (sabeurthabti) [inline package.json]: is prettier configured differently on my editor ?

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32945

**Comment by sabeurthabti** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32946

File: package.json [hunk 2]
```diff
@@ -29,16 +27,38 @@
...
+    ],
+    "coveragePathIgnorePatterns": [
+      "/node_modules/",
+      "/dist/"
+    ],
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/dist/"
+    ],
     "automock": false,
```
Comment (amahalwy) [inline package.json]: Test

**Comment by sabeurthabti** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32952

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33022

**Comment by AmmarHasan** [comment]
@naveed-ahmad @sabeurthabti  I am not able to use keyboard shortcuts to play/pause, play next ayah or play previous ayah

**Comment by naveed-ahmad** [comment]
@AmmarHasan keyboard shortcuts are coming soon iA.

**Pull request merged**